applications/tari_app_grpc/proto/types.proto

```diff
@@ -89,10 +89,8 @@ message HistoricalBlock {
     // The number of blocks that have been mined since this block, including this one. The current tip will have one
     // confirmation.
     uint64 confirmations = 1;
-    // An array of commitments of the outputs from this block that have subsequently been spent.
-    repeated bytes spent_commitments = 2;
     // The underlying block
```

Comment:
[Don't renumber the codes](https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers). You  remove it, or mark it as deprecated, but you must not change the numbers.